ibm_cloud_sdk_core/authenticators/basic_authenticator.py

```diff
@@ -39,8 +39,9 @@ class BasicAuthenticator(Authenticator):
     def __init__(self, username: str, password: str):
         self.username = username
         self.password = password
+        self.authorization_header = None
```

Comment:
Just a nit, but we don't need to initialize this to `None` since you're initializing it below :)

Reply:
Whoops you're right. I need to fix this.